src/scene/materials/standard-material.js

```diff
@@ -109,6 +109,13 @@ let _params = new Set();
  * are specularTint are set, they'll be multiplied by vertex colors.
  * @property {string} specularVertexColorChannel Vertex color channels to use for specular. Can be
  * "r", "g", "b", "a", "rgb" or any swizzled combination.
+ * @property {boolean} useSpecularityFactor Use the specularity factor to weight specularity
+ * @property {number} specularityFactor The factor of specularity, exclusive to specular
+ * @property {Texture|null} specularityFactorMap The factor of specularity as a texture
+ * @property {boolean} f0Tint Multiply F0 reflection color
+ * @property {Vec3} f0 The F0 color
+ * @property {Texture|null} f0Map The map used for the F0 reflection color
+ * @property {string} f0MapChannel Color channels of the F0 color
```

Comment:
```suggestion
 * @property {boolean} f0Tint Multiply F0 reflection color.
 * @property {Vec3} f0 The F0 color.
 * @property {Texture|null} f0Map The map used for the F0 reflection color.
 * @property {string} f0MapChannel Color channels of the F0 color.
```

Reply:
Maybe explain what F0 is somewhere? Also, consider adding 'Defaults to...' to each property description.